### DISCUSSION TITLE
How can i get the data from my contact form in my strapi controller with the create function?

### DISCUSSION THREAD

**Discussion by tom-baboons** [strapi/strapi#8026] [Legacy Topics]
Hi,

On create I want to send a confirmation mail with SendGrid. That succeed when I set a static mail. But when I want to get the email from the form I don't get a response in the data.

```
module.exports = {
  async create(ctx) {
    try {
      const send = await strapi.plugins['email'].services.email.send({
        to: 'test@test.com',
        subject: 'Use strapi email provider successfully',
        text: 'Hello world!',
        html: 'Hello world!',
        template_id: 'd-a5543b117edb4b51baf95dfb4ff303fa',
      });
    } catch (error) {
      console.log(error);
      this.error = error
    }
  }
};
```

**Comment by derrickmehaffy** [comment]
You will want to check the `ctx.request.body` for a POST request that contains all the information sent in the request body. Assuming the object key would `email` you should be able to get it from `ctx.request.body.email`

**Reply by tom-baboons** [reply]
Great! This one work. Thank you!
